doc/modules/classification_threshold.rst

```diff
@@ -1,6 +1,6 @@
 .. currentmodule:: sklearn.model_selection
 
-.. _TunedThresholdClassifierCV:
+.. _threshold_tunning:
 
 ==================================================
 Tuning the decision threshold for class prediction
```

Comment:
@glemaitre I wonder if it would be interesting to add another scenario where threshold tunning may be of interest; imbalanced datasets?